pkg/trace/writer/trace.go

```diff
@@ -110,7 +110,7 @@ func NewTraceWriter(
 	}
 	climit := cfg.TraceWriter.ConnectionLimit
 	if climit == 0 {
-		climit = 100
+		climit = 5
```

Comment:
We lose the 4 to 1 ratio between stats and traces, is that a risk?

Reply:
There's a small risk here, but I think with benchmarking / testing we can confirm if there are any issues and make adjustments before going to `main`